Install/RevitIFCSetupWix/Product.wxs

```diff
@@ -129,12 +129,12 @@
             </Component>
          <?endif?>
 
-         <Component Id = "SteelCon_ASIFC_DLL" Guid="951F4BE6-CCA7-45B3-B687-3A82248B2F70">
+         <!--<Component Id = "SteelCon_ASIFC_DLL" Guid="951F4BE6-CCA7-45B3-B687-3A82248B2F70">

```

Comment:
We have clearance to publish the ASIFC dll now, but we need to build it specially. What I did to 21.3.0.0 (that is withdrawn) is to rebuild ASIFC in Release 2021 but with IFC dlls from the open source.
There are a few changes needed in the code (one is in Exporter.cs) that have codes commented to work with this change.